server/routes/controllers/getMyBookings.js

```diff
@@ -0,0 +1,15 @@
+const models = require('../../models/index.js');
+
+const getMyBookingsRoute = (req, res) => {
+  return models.getMyBookings()
+    .then((bookings) => {
+      console.log('CONTROLLER: MY BOOKINGS', bookings)
+      res.send(bookings)
```

Comment:
should you set this to status code 200 here?

Reply:
@matthewwrobel Absolutely right. Let me make that change.